src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -28,9 +28,13 @@
         "portable-net45+win8"
       ]
     },
-    "net45": {
+    "net46": {
```

Comment:
I changed the nuget package's folder name and adjusted build targets to account for it. I haven't re-targeted all of our desktop assets to net46 unless there was a need, but we can do that in the future without problems now.